x-pack/plugins/security_solution/public/entity_analytics/components/entity_store/components/dashboard_panels.tsx

```diff
@@ -0,0 +1,246 @@
+          </EuiFlexItem>
+          <EuiFlexItem>
+            <EntitiesList />
+          </EuiFlexItem>
+        </>
+      )}
+      {entityStore.status === 'enabled' && !isRiskScoreAvailable && (
+        <>
+          <EuiFlexItem>
+            <EnableEntityStore
+              onEnable={() => setModalState({ visible: true })}
+              loadingRiskEngine={riskEngineInitializing}
+              enablements="riskScore"
+            />
+          </EuiFlexItem>
+
+          <EuiFlexItem>
+            <EntitiesList />
+          </EuiFlexItem>
+        </>
+      )}
+
+      {entityStore.status === 'not_installed' && !isRiskScoreAvailable && (
+        // TODO: Move modal inside EnableEntityStore component, eliminating the onEnable prop in favour of forwarding the riskScoreEnabled status
+        <EnableEntityStore
+          enablements="both"
+          onEnable={() => setModalState({ visible: true })}
+          loadingRiskEngine={riskEngineInitializing}
+        />
+      )}
+
+      {entityStore.status === 'not_installed' && isRiskScoreAvailable && (
+        <>
+          <EuiFlexItem>
+            <EnableEntityStore
+              enablements="store"
+              onEnable={() =>
+                setModalState({
+                  visible: true,
+                })
+              }
+            />
+          </EuiFlexItem>
+          <EuiFlexItem>
+            <EntityAnalyticsRiskScores riskEntity={RiskScoreEntity.user} />
+          </EuiFlexItem>
+          <EuiFlexItem>
+            <EntityAnalyticsRiskScores riskEntity={RiskScoreEntity.host} />
+          </EuiFlexItem>
+        </>
+      )}
+
+      <EntityStoreEnablementModal
+        visible={modal.visible}
+        toggle={(visible) => setModalState({ visible })}
+        enableStore={enableEntityStore}
+        riskScore={{ disabled: isRiskScoreAvailable, checked: !isRiskScoreAvailable }}
+        entityStore={{
+          disabled: entityStore.status === 'enabled',
+          checked: entityStore.status !== 'enabled',
+        }}
+      />
+    </EuiFlexGroup>
+  );
+};
+
+interface EnableEntityStoreProps {
+  onEnable: () => void;
+  enablements: 'store' | 'riskScore' | 'both';
+  loadingRiskEngine?: boolean;
+}
+
+export const EnableEntityStore: React.FC<EnableEntityStoreProps> = ({
+  onEnable,
+  enablements,
+  loadingRiskEngine,
+}) => {
+  const title =
+    enablements === 'store'
+      ? ENABLE_ENTITY_STORE_TITLE
+      : enablements === 'riskScore'
+      ? ENABLE_RISK_SCORE_TITLE
+      : ENABLE_ALL_TITLE;
+
+  if (loadingRiskEngine) {
+    return (
+      <EuiPanel hasBorder>
+        <EuiEmptyPrompt
+          icon={<EuiLoadingLogo logo="logoElastic" size="xl" />}
+          title={<h2>{ENABLEMENT_INITIALIZING_RISK_ENGINE}</h2>}
+        />
+      </EuiPanel>
+    );
+  }
+  return (
+    <EuiEmptyPrompt
+      css={{ minWidth: '100%' }}
+      hasBorder
+      layout="horizontal"
+      className="eui-fullWidth"
+      title={<h2>{title}</h2>}
```

Comment:
are minWidth and eui-fullWidth both needed?

Reply:
yeah, without the `min-width` it doesnt stretch enough. 
I actually think the real fix is to somehow let the `body` stretch as much as possible, but I don't see how to do that with this `EuiEmptyPrompt`.
This component seems to be what Pavel based the UI off in figma, but there's a few quirks with it. It might be necessary to just use some other Eui stuff like `EuiPanel`